TrustSDK/Classes/Signer/SigningOutput.swift

```diff
@@ -0,0 +1,13 @@
+//  Copyright © 2018 Trust.
+//
+//	This file is part of TrustSDK. The full TrustSDK copyright notice, including
+//	terms governing use, modification, and redistribution, is contained in the
+//	file LICENSE at the root of the source code distribution tree.
+	
+
+import Foundation
+import TrustWalletCore
+
+public protocol SigningOutput { }
```

Comment:
There is a typealias in TrustWalletCore for `SigningInput`, I think we can do same for `SigningOutput`

Reply:
Can you create an issue for that?

Reply:
will do